sbnobj/Common/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
+add_subdirectory(Analysis)
```

Comment:
Is the inclusion of this line an error? I guess it's personal code that you don't want merged?

Reply:
That's the existing [`sbnobj/Common/Analysis` directory](https://github.com/SBNSoftware/sbnobj/tree/develop/sbnobj/Common/Analysis) — just moved up from line 3 to alphabetize for readability.

Reply:
#headdesk